theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx

```diff
@@ -0,0 +1,476 @@
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+<u>Note</u>: It's possible to replace the `+` with a particular version of THEOplayer that should be used, for example:
+
+`implementation 'com.theoplayer.theoplayer-sdk-android:core:8.0.0'`
+
+## Setting up the license
+
+You need to configure your THEOplayer license, if you don't enter this license string, you might encounter licensing issues (e.g.: when trying to play non-whitelisted video streams).
+
+First, obtain a THEOplayer Android SDK license through the [THEOplayer Developer Portal](https://portal.theoplayer.com/login).
+![](https://cdn.theoplayer.com/images/git/theoplayer-android-sdk-license-string.png)
+
+Then, identify your license string from the screenshot above and add the `<meta-data ...` tag below at the appropriate location in your `AndroidManifest.xml`, and swap `your_license_here` with your
+actual license string.
+
+```xml
+<application>
+    <meta-data
+        android:name="THEOPLAYER_LICENSE"
+        android:value="your_license_here" />
+</application>
+```
+
+Additionally, to play online videos with THEOplayer, you first need to add [INTERNET](https://developer.android.com/reference/android/Manifest.permission.html#INTERNET) permission into the `AndroidManifest.xml` file too.
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+More information regarding the license can be found [here](../../../how-to-guides/12-license/00-introduction.md).
+
+## Add THEOplayer view to your application
+
+THEOplayer can be added to an application either through a layout (i.e. XML) or programmatically through the constructor API. This guide uses the layout approach. Refer to [Using the constructor API](https://github.com/THEOplayer/theoplayer-sdk-android#b-using-the-constructor-api)
+if you want to use the latter approach. This approach also allows you to configure the license string through a programmatic configuration instead of the `AndroidManifest.xml`.
+
+You will add some code to the activity_main.xml file.
+
+![](../../../assets/img/android-getting-started-13_2.png)
+
+You can add the `THEOplayerView` to the layout of the `MainActivity` with the snippet below.
+
+At least the `id`, `width` and `height` attributes need to be specified on the new view. The `id` is important as it is used to identify the view object in your `MainActivity` class.
+
+```xml
+<com.theoplayer.android.api.THEOplayerView
+    android:id="@+id/theoplayer"
+    android:layout_width="match_parent"
+    android:layout_height="300dp"
+    app:layout_constraintTop_toTopOf="parent">
+</com.theoplayer.android.api.THEOplayerView>
+```
+
+![](../../../assets/img/android-getting-started-14_2.png)
+
+## Use THEOplayer API in your application
+
+Now in the `MainActivity` we can retrieve the `THEOplayerView` by the `id` we set earlier using the `findViewById` method.
+
+#### 1. Get the THEOplayer instance
```

Comment:
These headings are nested one level too deep.
```suggestion
### 1. Get the THEOplayer instance
```
Same for the other numbered steps.

Reply:
Done